nrf_modem/include/nrf_socket.h

```diff
@@ -909,90 +802,79 @@ int nrf_accept(int sock, void *p_cliaddr, nrf_socklen_t *p_addrlen);
  */
 
 /**
- * @brief Function for converting a human-readable IP address to a form usable by the socket API.
+ * @brief Convert IPv4 and IPv6 addresses between binary and text form
  *
- * @details This function will convert a string form of addresses and encode it into a byte
- *          array in network byte order.
- *
- * @note    Currently not supporting mixed IPv4 and IPv6 format strings.
- *
- * @param[in]  family  Address family. NRF_AF_INET or NRF_AF_INET6.
- * @param[in]  p_src   Null-terminated string containing the address to convert.
- * @param[out] p_dst   Pointer to a struct nrf_in_addr or nrf_in6_addr where the address will
- *                     be stored.
- *
- * @return 1 on success
- *         0 if p_src does not contain a valid address,
- *         -1 on error, and errno indicates the reason for failure.
+ * @details
+ * @rst
+ * See `POSIX.1-2017 article
+ * <https://pubs.opengroup.org/onlinepubs/9699919799/functions/inet_ntop.html>`__
```

Comment:
```suggestion
 * <https://pubs.opengroup.org/onlinepubs/9699919799/functions/inet_pton.html>`__
```

Reply:
The pages (pton and ntop) contain similar information. However, they are not completely identical. If this is supposed to be `ntop`, there should be a note.

Reply:
Updated